client/state/sites/test/selectors.js

```diff
@@ -1184,4 +1194,675 @@ describe( 'selectors', () => {
 			expect( frontPageType ).to.eql( 'page' );
 		} );
 	} );
+
+	describe( '#canJetpackSiteManage()', () => {
+		it( 'it should return `null` for a non-existing site', () => {
+			const state = {
```

Comment:
I don't think we should be too afraid of duplicating code in tests. It indeed makes the test longer but it gives all the information of the input (and the expected output) at the same place, which is good imo.